mage_ai/api/resources/PipelineRunResource.py

```diff
@@ -54,11 +56,33 @@ def collection(self, query_arg, meta, user, **kwargs):
 
         results = (
             PipelineRun.
-            query.
-            options(selectinload(PipelineRun.block_runs)).
```

Comment:
I think it breaks the block_runs and block_runs_count fields in the pipeline_run list response.
https://github.com/mage-ai/mage-ai/blob/master/mage_ai/api/presenters/PipelineRunPresenter.py#L28-L29

Reply:
K will fix